gluster-exporter/metric_volume.go

```diff
@@ -74,6 +156,63 @@ func healCounts() error {
 	return nil
 }
 
+func getVolumeProfileInfoLabels(volname string, brick string) prometheus.Labels {
+	return prometheus.Labels{
+		"volume": volname,
+		"brick":  brick,
+	}
+}
+
+func getVolumeProfileFopInfoLabels(volname string, brick string, fop string) prometheus.Labels {
+	return prometheus.Labels{
+		"volume": volname,
+		"brick":  brick,
+		"fop":    fop,
+	}
+}
+
+func profileInfo() error {
+	isLeader, err := gluster.IsLeader()
+	if err != nil {
+		log.WithError(err).Error("Unable to find if the current node is leader")
+		return err
+	}
+	if !isLeader {
+		return nil
+	}
+
+	volumes, err := gluster.VolumeInfo()
+	if err != nil {
+		return err
+	}
+	for _, volume := range volumes {
+		name := volume.Name
+		profileinfo, err := gluster.VolumeProfileInfo(name)
```

Comment:
Doesn't this assume volume profile is started ? Is that the default for deployment?

Reply:
I understand `VolumeProfileInfo` would return an error in case profiling is not enabled. @aravindavk is it true?

Reply:
We can add intelligence to enable the Volume profile if profile collector is enabled in exporter config file. You can open new issue for the same if not planning to address in this PR

Reply:
@aravindavk Ack. Will create an issues for the same.